crates/eww/src/widgets/systray.rs

```diff
@@ -188,3 +189,18 @@ impl Item {
         }
     }
 }
+
+async fn load_icon_for_item(icon: &gtk::Image, item: &notifier_host::Item, size: i32, scale: i32) {
```

Comment:
could you leave a comment as to how this supports HiDPI, and why we can't just set_from_pixbuf?

Reply:
> could you leave a comment as to how this supports HiDPI,

see https://github.com/ralismark/eww/pull/3#issuecomment-1697081023
> and why we can't just set_from_pixbuf?

Because GTK3 doesn't provide an equivalent API that allows us to pass a custom scaling factor xD, check this SO answer https://stackoverflow.com/questions/56531363/gtk-image-on-hidpi-displays